Testscripts/Linux/nested_vm_utils.sh

```diff
@@ -63,6 +62,12 @@ Install_KVM_Dependencies()
         Update_Test_State $ICA_TESTFAILED
         exit 0
     fi
+    if [ $DISTRO_NAME == "sles" ]; then
+        add_sles_network_utilities_repo
+        echo "Install aria2 in $DISTRO_NAME"
+        zypper --non-interactive --no-gpg-checks in aria2
```

Comment:
if the distro is sles, in install_package it will run zypper install again, so there's no need to run it here